ui/library/src/context/ScrollContext.ts

```diff
@@ -159,21 +171,29 @@ export function useScrollContextProps(): IScrollContext {
       ?.scrollIntoView({ behavior: 'smooth' });
   }, []);
 
+  const getVisibleElement = (visibleElements: Map<any, number>): any => {
```

Comment:
Better function name such as `getMaxVisibleElement`; Use a specific type for Map instead of `any`

Reply:
✅